src/main/kotlin/io/github/mojira/arisa/ArisaMain.kt

```diff
@@ -46,7 +47,25 @@ fun main() {
     val failedTickets = mutableSetOf<String>()
 
     val cache = QueryCache()
-    var moduleExecutor = ModuleExecutor(jiraClient, config, cache)
+
+    val helperMessagesFile = File("helper-messages.json")
+    val helperMessages = HelperMessages.fetch().fold(
+        {
+            HelperMessages.deserialize(
+                if (helperMessagesFile.exists()) {
+                    helperMessagesFile.readText()
+                } else {
+                    """{"variables":{},"messages":{}}"""
+                }
+            )!!
+        },
+        {
+            helperMessagesFile.writeText(it.serialize())
+            it
+        }
+    )
```

Comment:
If something gets merged in helper-messages, does arisa notice that change with this or does it only fetch once on startup?

Reply:
It only fetches once on startup  
The file serves as a fallback in case the bot cannot download the file from GitHub for some reasons.

Reply:
It would be better if Arisa became aware of changes to Helper messages at least after some time. The most feasible way to implement this would probably be to re-fretch in a certain (configurable?) interval.
But for the sake of this pr, we can move that to a separate issue for future work.